app/models/manageiq/providers/cloud_automation_manager/configuration_manager/refresh_parser.rb

```diff
@@ -27,17 +27,18 @@ def configuration_profile_inv_to_hashes(profiles)
         end
       end
 
+      
       def configured_system_inv_to_hashes(configured_systems)
         configured_systems.to_a.collect do |cs|
           {
             :type         => "ManageIQ::Providers::CloudAutomationManager::ConfigurationManager::ConfiguredSystem",
             :manager_ref  => cs["id"].to_s,
             :hostname     => cs["name"],
-            :last_checkin => cs["last_compile"],
-            :build_state  => cs["build"] ? "pending" : nil,
-            :ipaddress    => cs["ip"],
-            :mac_address  => cs["mac"],
-            :ipmi_present => cs["sp_ip"].present?,
+            :last_checkin => cs["created_at"],
```

Comment:
For Foreman this is the last time a system checked in with the puppet master.  Is this a one time interaction, or is there a better field to look at for recurring interactions between the server and the configured system?

Reply:
In terraform you can do updates to deployed systems - the created date will change based on updates. There is a lot more information we want to capture and I am trying to hammer some of that data in the configured system schema you are using. We will definitely need to revisit it.